src/main/perl/RuleBasedEditor.pm

```diff
@@ -0,0 +1,1151 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+=pod
+
+=head1 DESCRIPTION
+
+This module implements a rule-based editor that is used to modify the content
+of an existing file without taking care of the whole file. Each rule
+driving the edition process is applied to all matching lines. The input for
+updating the file is the Quattor configuration and conditions can be defined
```

Comment:
`Quattor configuration` makes no sense in CAF, you probably mean a hashref or something else